jobs/common/scripts/install-go.sh

```diff
@@ -24,7 +24,7 @@ case $(uname -m) in
     ;;
 esac
 
-if [[ -z "${GOVERSION}" ]]; then
+if [[ "${GOVERSION}" == "\'\'" ]]; then
```

Comment:
We have `set -eux` at the top of the bash script. if ${GOVERSION} doesn't exist at all, I'm pretty sure the script just fails with an undefined environment varible, doesn't it?